### DISCUSSION TITLE
Testing with Giga R1 Wifi

### DISCUSSION THREAD

**Discussion by j-davis-2022** [pschatzmann/arduino-audio-tools#1741] [Ideas]
I saw in AudioConfig.h that you have drafted support for the Giga R1 Wifi, but it has not been tested. I have a Giga I'm hoping to use with your arduino-espeak-ng repository. If you're okay with the idea, I'd love to help test the Giga for the wiki/documentation.

**Comment by pschatzmann** [comment]
Feel free to try it out. 
I think there is no I2S support yet so https://github.com/pschatzmann/stm32-i2s might need to be extended.
But you can always use VS1053 via SPI

ps.
I actually prefer [this much cheaper option](https://www.pschatzmann.ch/home/2024/07/25/the-power-of-stm32h7-microcontrollers/)!
